docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst

```diff
@@ -1115,63 +1115,165 @@ For the purposes of our simulation, we made a few assumptions:
+  indicates the child has not yet started eating fortified food).
+  """
+    return (0                         if time_since_fortified < 0   else
+            time_since_fortified/0.5  if time_since_fortified < 0.5 else
+            1)
+
+.. Note::
+
+  Since children don't start eating fortified food until 6 months of age, the
+  argument ``time_since_fortified`` for the time-lag function will be *either*
+  the time since the child's household got fortification coverage, *or* the time
+  since the child turned 6 months old, *whichever is smaller*.
+
+.. Note::
+
+  When calibrating hemoglobin levels for baseline coverage (see the
+  :ref:`baseline calibration <baseline_calibration_hb_bw_section>` section
+  below), we will use **only** the age effect to compute the treatment-deleted
+  hemoglobin levels of the population (the down-shift), whereas we will use
+  **both** the age effect *and* the time-lag effect when applying the treatment
+  to an individual simulant (the up-shift). This will cause our population's
+  overall mean hemoglobin level to be slightly lower than the value in GBD, but
+  not by much, and this is the simplest strategy that seems reasonable.
+
+.. Note::
+
+  When calibrating hemoglobin levels for baseline coverage (see the
+  :ref:`baseline calibration <baseline_calibration_hb_bw_section>` section
+  below), it is *possible* (but highly unlikely with the shift size we're dealing with) that a simulant's hemoglobin level can end up *negative* after the down-shift. **In the case of a negative hemoglobin level, we will arbitrarily reset the hemoglobin level to 10 g/L**.
+
+.. Note::
 
-	# at the individual simulant level
-	if age_i < 0.5:
-		md_i = 0
-	elif age_i < 2:
-		md_i = MD * (age_i - 0.5) / 1.5
-	else:
-		md_i = MD
+  As described in the :ref:`Dietary Iron Deficiency / Iron Deficiency
+  Documentation <2017_cause_iron_deficiency>`, the hemoglobin shift from iron
+  fortification should be applied **only** to simulants who are *iron
+  responsive*. This applies both to the baseline calibration (down-shift) and
+  the application of the effect size to simulants with fortification coverage
+  (up-shift).
 
-The effect size of the iron fortification on a simulant's hemoglobin level for
-new **intervention** coverage of iron fortification should be determined as
-follows:
+.. Note::
+
+  When determining whether a simulant gets covered by our iron fortification
+  program, the child's propensity for coverage (for the hemoglobin effect)
+  should be **the same** as their mother's propensity for coverage (for the
+  birthweight effect). In other words, when comparing the random propensity
+  against the iron fortification :ref:`coverage function
+  <coverage_algorithm_section>`, the same random number should be used to
+  determine whether the child's mother ate fortified food during pregnancy as is
+  used to determine whether the child eats fortified food.
```

Comment:
👍  but we could consider making them positively correlated and not identical in a future iteration.  Maybe a sentence in a limitations section.